source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/overview/products/item/index.tsx

```diff
@@ -13,6 +13,8 @@ import ItemActions from './actions';
 import ProductIcon from '@ui/components/product-icon';
 import TimezoneLabel from '@ui/components/labels/timezone';
 import { withTranslations, i18nProps } from '@lib/i18n';
+import { isEmpty } from '@lib/collection';
+import LaunchIcon from '@material-ui/icons/Launch';
 
```

Comment:
Should there an order to imports?

Reply:
I generally put all the external library imports at the top, do an empty line, and then the project-local imports. I've been doing:
```
lib imports

from @data
from @lib
from @ui
```

don't know why that order for the project stuff, but it feels right

Reply:
@NullVoxPopuli any changes in the order of imports?

Reply:
not at the moment. if linting doesn't catch it, no point in spending time on it. (this'll be changing soon :) )